main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
Move this commit into a different branch, this is supposed to be a separate feature